crypto/evp/e_des3.c

```diff
@@ -354,6 +353,7 @@ static int des_ede3_unwrap(EVP_CIPHER_CTX *ctx, unsigned char *out,
 
     if (!CRYPTO_memcmp(sha1tmp, icv, 8))
         rv = inl - 16;
+    OPENSSL_cleanse(tmp, 8);
     OPENSSL_cleanse(icv, 8);
     OPENSSL_cleanse(sha1tmp, SHA_DIGEST_LENGTH);
     OPENSSL_cleanse(iv, 8);
```

Comment:
On **side** note, [i.e. no action is actually expected,] it wouldn't be inappropriate to coalesce OPENSSL_cleanse chains like this by
```
struct { unsigned char tmp[8], icv[8], etc.; } scratch;
...
memcpy(scratch.tmp, ...
...
OPENSSL_cleanse(&scratch, sizeof(scratch));
```